RHEL/6/input/checks/ftp_limit_users.xml

```diff
@@ -0,0 +1,46 @@
+<def-group>
+  <definition class="compliance" id="ftp_limit_users" version="1">
+    <metadata>
+      <title>Limit Users Allowed to Access FTP</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
+      </affected>
+      <description>Only authorized users should be able to access FTP.</description>
+      <reference source="galford" ref_id="20140812" ref_url="test_attestation" />
+    </metadata>
+    <criteria comment="FTP is not being used or the conditions are met" operator="OR">
+      <extend_definition comment="vsftp package is not installed" definition_ref="package_vsftpd_installed" negate="true" />
+      <criteria comment="FTP configuration conditions are not set or are met" operator="AND">
+        <criterion comment="Enable ftp user list" test_ref="test_ftp_user_list_enable" />
+        <criterion comment="Set ftp user list file" test_ref="test_ftp_user_list_file" />
+        <criterion comment="Use ftp user list" test_ref="test_ftp_user_list_deny" />
+      </criteria>
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="Enable ftp user list" id="test_ftp_user_list_enable" version="1">
+    <ind:object object_ref="object_ftp_user_list_enable" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object comment="Enable ftp user list" id="object_ftp_user_list_enable" version="1">
+    <ind:filepath>/etc/vsftpd/vsftpd.conf</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*userlist_enable[\s]*=[\s]*YES$</ind:pattern>
+    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
+  </ind:textfilecontent54_object>
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="Set ftp user list file" id="test_ftp_user_list_file" version="1">
+    <ind:object object_ref="object_ftp_user_list_file" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object comment="Set ftp user list file" id="object_ftp_user_list_file" version="1">
+    <ind:filepath>/etc/vsftpd/vsftpd.conf</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*userlist_file[\s]*=[\s]*\/etc\/vsftp\.ftpusers$</ind:pattern>
```

Comment:
This is not a blocker (once all the already listed issues are fixed). But maybe in the 2-nd version of this check we wouldn't insist the file needs to be exactly ```/etc/vsftp.ftpusers``` file (it can be any filename). And we could hopefully verify via OVAL particular file exists on the system and has some usernames listed in it (entries separated by newline).

But this can wait for 2-nd script / OVAL check version. It's not a blocker.